full-v3.yml

```diff
@@ -1365,6 +1582,16 @@ definitions:
       id:
         type: string
         x-validation: true
+      lms_state:
+        type: string
+        x-nullable: true
+        x-validation: true
+        enum:
+        - "matching_in_progress"
+        - "error"
+        - "disconnected"
+        - ""
```

Comment:
More a curiosity question--in what cases would we get an empty string?

Reply:
They shouldn't get an empty string but I think we've added it as an option in the past just to cover our bases, so just following the pattern here